intellij-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -6,13 +6,8 @@
     <description><![CDATA[Arcs manifest support]]></description>
 
     <extensions defaultExtensionNs="com.intellij">
-        <fileTypeFactory implementation="com.google.arcs.ArcsFileTypeFactory"/>
-        <lang.parserDefinition language="Arcs" implementationClass="com.google.arcs.ArcsParserDefinition"/>
-        <lang.syntaxHighlighterFactory language="Arcs" implementationClass="com.google.arcs.ArcsSyntaxHighlighterFactory"/>
-    </extensions>
-
-    <extensions defaultExtensionNs="com.intellij">
-        <fileType name="Arcs manifest file" implementationClass="com.google.arcs.ArcsFileType"
```

Comment:
I think we want to keep this too, this is the "new way" to do it for 2019.02 and later, the other way with the file type factory is the old way.

See the `NOTE` section [here](https://www.jetbrains.org/intellij/sdk/docs/reference_guide/custom_language_support/registering_file_type.html).

The versioning/compatibility and code, and plugin.xml ramifications, are really complicated across IntelliJ and Android Studio. It seems there are legacy extentions/apis/configs, and new ones.

For example a lot of the documentation [such as here](https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html#modules-available-in-all-products) also seems to say that plugin.xml should include:

```
<depends>com.intellij.modules.platform</depends>
<depends>com.intellij.modules.lang</depends>
```

It seems to work without it, but I suspect it could be prone to breaking, or may not work without it in Android Studio, etc. @piotrswigon and I had a series of changes to `plugin.xml` to get it working in AS and I am not sure which ones were actually critical.

Reply:
After all the changes I've reverted them and confirmed that it works in AS the way it was. The problem was the Blaze plugin all along.

Still, these changes suggested could be right and Intellij could be lenient in allowing it to run / access deps.

Reply:
See https://intellij-support.jetbrains.com/hc/en-us/community/posts/360004382559-fileTypeFactory-to-fileType-correct-usage-and-compatibility-in-IDEA-2019-2. I think the old way maintains better compatibility.

I am not sure what experiments you did with plugin.xml. I had an issue which I can reproduce 100% on both the ASWB and the normal Android Studio on my Mac: a clean install of this plugin won't associate itself with *.arcs files and if you manually associate, it will forget after the IDE is restarted.

This PR fixes this issue.